documentation/src/main/asciidoc/topics/upgrading.adoc

```diff
@@ -1,5 +1,16 @@
 = Upgrading from 10.0 to 10.1
 
+== Maximum Idle Timeouts with Clustered Cache Modes
+Maximum idle expiration has been changed to improve data consistency with clustered cache modes when Infinispan nodes fail.
```

Comment:
Sorry to keep beating this dog, but should we mention that performance will be slower?